package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vectorcasttestexplorer",
   "displayName": "VectorCAST Test Explorer",
   "description": "VectorCAST Test Explorer for VS Code",
-  "version": "1.0.12",
```

Comment:
This is wrong.

Reply:
The current release is 10 so 11 will be next.

Reply:
Oh, who moved it to 12?!

Reply:
Ah, https://github.com/vectorgrp/vector-vscode-vcast/pull/86